Rubberduck.Resources/Inspections/InspectionInfo.resx

```diff
@@ -349,4 +349,7 @@ If the parameter can be null, ignore this inspection result; passing a null valu
   <data name="AssignmentNotUsedInspection" xml:space="preserve">
     <value>An assignment is immediately overridden by another assignment or is never referenced.</value>
   </data>
+  <data name="ExcelMemberMayReturnNothingInspection" xml:space="preserve">
+    <value>A Function or Property returning a reference type that can be 'Nothing' should not have the return value used with a test for 'Is Nothing'. If the member does not return a valid object, attempting to use the return value will result in a run-time error 91 - "Object variable or With block variable not set".</value>
```

Comment:
"Should *not* have the return value used with a test for 'Is Nothing'"? :confused:

Reply:
That should be "without", although I'm not sure how happy I am with this in general - it seems overly wordy.

Reply:
Does it *always* end in RTE 91? I'm wondering if there are other uses where we would get a different RTE with different syntax besides member access. 

We could try this wording:

`A procedure that returns an object may return 'Nothing'. That will cause runtime errors in subsequent member access. Perform a 'Is Nothing' check after the 'Set' assignment to guard against runtime errors.`